[docs] VACMS-11966: facility runbook enhancements (#12860)
department-of-veterans-affairs/va.gov-cms: .github/ISSUE_TEMPLATE/runbook-vamc-facility-name-change.md

- [ ] Name of submitter (if applicable)
Submitter: <insert_name>

- [ ] If the submitter is an editor, send them links to any relevate KB articles for the VAMC product. Let them know that facility changes can take between 75 days and 4 months after submitting a request, according to VAST administrators.
KB articles: <insert_kb_article_links>

- [ ] Link to facility in production:
Facility link: <insert_facility_link>

## Acceptance criteria

## VAMC Facility name change

#### CMS help desk steps
- [ ] 1. The title change comes from Lighthouse to Drupal.
- [ ] 2. Coordinate with Facilities team to create a [redirect request](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/new?assignees=&labels=Redirect+request&template=redirect-request-facility-url.md&title=Redirect+Request+for%3A+%3Cinsert+facility+name%3E) from the original URL for the facility to the new URL for the facility.

<insert_redirect_request_link>

#### CMS engineer steps
- [ ] 3. CMS engineer locates the newly renamed VAMC Facility (https://prod.cms.va.gov/admin/content/bulk) Search by new name
- [ ] 4. CMS engineer updates URL alias for this facility
- [ ] 5. CMS engineer resaves this facility

> - [ ] 6. CMS engineer makes bulk alias changes to facility service nodes. (https://prod.cms.va.gov/admin/content/bulk?type=health_care_local_health_service)

Reviewer: ```suggestion
- [ ] 6. CMS engineer makes bulk alias changes to the facility service nodes. (https://prod.cms.va.gov/admin/content/bulk?type=health_care_local_health_service)
```

> - [ ] 7. CMS engineer bulk saves fixed titles to facility service nodes. (https://prod.cms.va.gov/admin/content/bulk?type=health_care_local_health_service)

Reviewer: ```suggestion
- [ ] 7. CMS engineer bulk saves the fixed titles to facility service nodes. (https://prod.cms.va.gov/admin/content/bulk?type=health_care_local_health_service)
```

> - [ ] 8. CMS engineer updates menu title for facility

Reviewer: ```suggestion
- [ ] 8. CMS engineer updates the menu title for the facility
```

- [ ] 9. CMS engineer may also need to directly edit the VAMC System menu to alpha sort the menu item after the title changes

> - [ ] 10. CMS engineer updates Alt text for facility image, if relevant

Reviewer: ```suggestion
- [ ] 10. CMS engineer updates the Alt text for the facility image, if relevant
```

- [ ] 11. CMS engineer updates Meta description (TBD: some backwards compatibility for SEM, by including something like ", formerly known as [previous name]".

> - [ ] 12. CMS engineer edit facility node and remove flag `Changed name` then save node

Reviewer: ```suggestion
- [ ] 12. CMS engineer edits the facility node and removes flag `Changed name` then saves node
```
